### PR TITLE
adds .appveryor.yml

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+environment:
+  matrix:
+    - nodejs_version: "4"
```
Comment (diasdavid) [inline .appveyor.yml]: No more need for Node.js 4

**Comment by diasdavid** [comment]
rebase master on this branch so that CI is green

**Comment by thisconnect** [comment]
Sorry I have to leave for today, should I be able to push `git push origin appveyor` (rebase noob)?

I did create the `appveyor` branch on github.com and edited the files through the web gui. For rebasing I tried

```bash
git clone https://github.com/ipfs/js-ipfsd-ctl.git
git checkout appveyor
git rebase master
git status
On branch appveyor
Your branch and 'origin/appveyor' have diverged,
and have 7 and 4 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
git push origin appveyor # wont let me
```

Else I'll fork again tomorrow and create a new PR from my fork

**Comment by diasdavid** [comment]
@thisconnect you needed to do `git rebase origin/master appveyor`. That's fine, I'll just merge this, it is a config file and not code. 

You will need to rebase for https://github.com/ipfs/js-ipfsd-ctl/pull/155

**Comment by thisconnect** [comment]
thanks!

**Comment by diasdavid** [comment]
Thank you for shipping this :)

**Pull request merged**